backtesting/backtesting.py

```diff
@@ -90,7 +90,9 @@ def I(self,  # noqa: E743
         same length as `backtesting.backtesting.Strategy.data`.
 
         In the plot legend, the indicator is labeled with
-        function name, unless `name` overrides it.
+        function name, unless `name` overrides it. If `func` returns
+        multiple arrays, `name` can be a collection of strings, and
+        the size must agree with the number of arrays returned.
```

Comment:
```suggestion
        multiple arrays, `name` can be a sequence of strings, and
        its size must agree with the number of arrays returned.
```
`set` is a standard "collection", yet its ordering is undefined (not even dict-like).